.github/ISSUE_TEMPLATE/release.md

```diff
@@ -22,7 +22,7 @@ title: 'k6 release v0.4x.0'
 - [ ] k6-docs PRs for all new or changed functionality have been created.
 - [ ] All PRs to the k6 repository in the current [milestone](https://github.com/grafana/k6/milestones) have been merged.
 - [ ] A pull request with the release notes and request the final reviews (including the @k6-browser, devrels folks).
-- [ ] Open a PR for bumping [the version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11)).
+- [ ] Open a PR for bumping [the version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11).
```

Comment:
To avoid my own confusion during this specific release, I would even rephrase:

```suggestion
- [ ] Open a separate PR for bumping [the version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11).
```

What do you think?

Reply:
You are unique with a real experience in following it so I value your feedback very highly.